src/Hosting/Hosting/test/Internal/HostingEventSourceTests.cs

```diff
@@ -107,6 +108,7 @@ public void HostStop()
 
         [Theory]
         [MemberData(nameof(RequestStartData))]
+        [Flaky("https://github.com/aspnet/AspNetCore-Internal/issues/2040", "All")]
```

Comment:
For whatever reason the `FlakyOn.All` constant wasn't resolving so I just passed in the actual value here